roles/openshift_hosted/defaults/main.yml

```diff
@@ -71,7 +71,8 @@ openshift_hosted_registry_name: docker-registry
 openshift_hosted_registry_wait: "{{ not (openshift_master_bootstrap_enabled | default(False)) }}"
 registry_volume_claim: 'registry-claim'
 openshift_hosted_registry_cert_expire_days: 730
-
+openshift_hosted_registry_external_gluster: False
+openshift_hosted_registry_storage_glusterfs_endpoints: 'glusterfs-registry-endpoints'
```

Comment:
This is taken care of in the filter plugin. You don't need to specify defaults here.